shap/explainers/_kernel.py

```diff
@@ -223,9 +226,10 @@ def shap_values(self, X, **kwargs):
             # vector-output
             s = explanation.shape
             if len(s) == 2:
-                outs = [np.zeros(s[0]) for j in range(s[1])]
+                outs = [np.zeros(s[0]) for _ in range(s[1])]
```

Comment:
Can we get a test to cover this part of the diff?

Reply:
That's a good point, will write tests for all those cases aswell